js/src/personal/maliha/Maliha.page.tsx

```diff
@@ -0,0 +1,47 @@
+import { Title, Text, Burger } from '@mantine/core'
+import { useDisclosure } from '@mantine/hooks'
+import styles from './Maliha.module.css'
+
+const Menu = () => {
```

Comment:
This is technically a new react component. For now, I'll enforce one react component per file. Could you move this into Menu.tsx and import it here?